lua/toggleterm/ui.lua

```diff
@@ -202,17 +202,29 @@ function M.switch_buf(buf)
   if cur_buf ~= buf then vim.cmd(fmt("keepalt buffer %d", buf)) end
 end
 
-local split_commands = {
-  horizontal = {
-    existing = "rightbelow vsplit",
-    new = "botright split",
-    resize = "resize",
-  },
-  vertical = {
-    existing = "rightbelow split",
-    new = "botright vsplit",
-    resize = "vertical resize",
-  },
+local function split_command(direction, existing)
+  if 'horizontal' == direction then
+    if existing == true then
```

Comment:
There is no need to directly compare against `true` this should just be
```suggestion
    if existing then
```
The same goes for all the other `variable == true` unless there is a specific need to worry about comparing with some other truthy value vs true, then just use `if variable then`